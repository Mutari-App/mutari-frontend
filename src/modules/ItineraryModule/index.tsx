export default function ItineraryModule() {
  return <div>ItineraryModule</div>
}
