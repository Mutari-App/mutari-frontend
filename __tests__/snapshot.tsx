import { render } from '@testing-library/react'
import ItineraryModule from '@/modules/ItineraryModule'

it('renders itinerary page unchanged', () => {
  const { container } = render(<ItineraryModule />)
  expect(container).toMatchSnapshot()
})
