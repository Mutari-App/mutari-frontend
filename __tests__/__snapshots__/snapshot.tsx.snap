// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders itinerary page unchanged 1`] = `
<div>
  <div>
    ItineraryModule
  </div>
</div>
`;
